apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,36 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import { FakeHttpService } from '../../data-access/fake-http.service';
+import { CardType } from '../../model/card.model';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [customClass]="'bg-blue-300'"
+      [list]="cities"
+      [type]="cardType"
+      customClass="bg-light-green">
+      <img style="width:200px;" showImg src="../../assets/img/city.png" />
+    </app-card>
+  `,
   standalone: true,
-  imports: [],
+  imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  constructor(
+    private http: FakeHttpService,
+    private cityStore: CityStore,
+  ) {}
 
-  ngOnInit(): void {}
+  cities: City[] = [];
+  cardType = CardType.CITY;
+
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((s) => this.cityStore.addAll(s));
+
+    this.cityStore.cities$.subscribe((s) => (this.cities = s));
```

Comment:
You don't need to subscribe two times, checkout [my solution](https://github.com/tomalaforge/angular-challenges/pull/576/files) to see a different way to get the data.